src/lava/lib/dl/slayer/loss.py

```diff
@@ -210,3 +210,54 @@ def forward(self, input, label):
                 label.flatten(),
                 reduction=self.reduction,
             )
+
+
+class DECOLLELoss(object):
```

Comment:
Let's move this class definition to `src/lava/lib/dl/decolle/loss.py`. Also remember to add `__init__.py` file to the folder.